templates/crate_details.hbs

```diff
@@ -44,10 +44,14 @@
               <ul class="pure-menu-list">
                 {{#each releases}}
                 <li class="pure-menu-item">
-                  {{#if this.build_status}}
-                  <a href="/crate/{{../name}}/{{this.version}}" class="pure-menu-link">{{this.version}}</a>
+                  {{#if this.yanked}}
+                  <a href="/crate/{{../name}}/{{this.version}}" class="pure-menu-link warn" title="{{../name}}-{{this.version}} is yanked"><i class="fa fa-fw fa-warning"></i> {{this.version}}</a>
```

Comment:
I can't help but thinking that this kind of crate-wide popup is something that could used for a lot of different things, it should probably get modularization out into a component sometime to prevent an eventual chain of `#if`s that all do essentially the same things

Reply:
Agreed, can you make an issue for that?

Reply:
Yeah, there's a bunch of redundancy in the current templates that probably should be pulled out into components, but I'm not sure exactly how to set that up with handlebars here.

Reply:
https://github.com/rust-lang/docs.rs/issues/740